privaterelay/middleware.py

```diff
@@ -56,7 +56,12 @@ def __call__(self, request):
         # in, use a server-side redirect to send them to the dashboard,
         # rather than handling that on the client-side:
         if request.path == "/" and settings.SESSION_COOKIE_NAME in request.COOKIES:
-            return redirect("accounts/profile/")
+            query_string = (
+                "?" + request.META["QUERY_STRING"]
+                if request.META["QUERY_STRING"]
+                else ""
+            )
```

Comment:
I was considering using [GET.urlencode](using [https://docs.djangoproject.com/en/3.2/ref/request-response/#django.http.QueryDict.urlencode]) instead:

```suggestion
            query_string = "?" + request.GET.urlencode() if request.GET else ""
```

However, I think there is no difference in performance or security. If we were also manipulating the querystring, I'd want to use the Python functions that work with the querystring, but we're not so `META["QUERY_STRING"]` is perfect.